src/scss/components/_modal.scss

```diff
@@ -0,0 +1,35 @@
+.bp-modal {
+  position: fixed;
+  top: 0; left: 0;
+  width: 100%; height: 100%;
+  z-index: 1000;
+
+  &__overlay {
+    width: 100%; height: 100%;
+    top: 0; left: 0;
+    position: absolute;
+    background: rgba(0, 0, 0, 0.7);
+    z-index: 1;
+  }
+
+  &__container {
+    position: relative;
+    z-index: 2;
+    background: #fff;
+    margin: 12*$m auto 0 auto;
+    max-width: 70%;
+    padding: 2*$m;
+
+    &__footer {
+      text-align: right;
+    }
+  }
+
+  &__cancel-button {
+    margin-right: 1.5*$m;
+  }
+}
+
+.bp-modal-template {
```

Comment:
Where is this class being used?

Reply:
This class is used to hide modal until trigger is fired.